internal/provider/provider.go

```diff
@@ -153,14 +154,23 @@ func azureProvider() *schema.Provider {
 				Description: "Allow OpenID Connect to be used for authentication",
 			},
 
+			// Azure CLI specific fields
+			"use_cli": {
```

Comment:
Would you please add the new attributes to the doc here: https://github.com/Azure/terraform-provider-azapi/blob/main/docs/index.md